docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -103,7 +103,7 @@
 
         - set_fact:
             helm_chart_overrides: "/usr/local/share/applications/overrides/wind-river-cloud-platform-deployment-manager-overrides-subcloud.yaml"
-            manager_chart: "/usr/local/share/applications/helm/wind-river-cloud-platform-deployment-manager-*.tgz"
+            manager_chart: {{ manager_chart }}
```

Comment:
This set fact is not necessary only if you want to set default value, but it was already set, let remove this line